docs/new_viz_guidance.md

```diff
@@ -35,14 +35,13 @@ Database are in the folder /queries/. These queries may be reused across multipl
 visualizations. When considering creating your own visualization, please
 see whether the available queries to our database instance are sufficient.
 
-If the existing queries won't suffice, the most straight-forward way to create
-your own is to copy one of the existing queries into the same directory with a
-descriptive filename:
+If the existing queries do not suffice:
+1. See if there is some minor changes or additional columns that could be added to
```

Comment:
"Exhaust the possibility of making minor changes, such adding new columns or aggregations, to existing queries."